ignite/services/chain/generate.go

```diff
@@ -178,6 +206,43 @@ func (c *Chain) Generate(
 		)
 	}
 
+	if targetOptions.isComposablesEnabled {
+		composablesPath = conf.Client.Composables.Path
+		if composablesPath == "" {
+			composablesPath = defaultComposablesPath
+		}
```

Comment:
We should make sure paths are absolute here and also for hooks, like it's done for typescript and Vuex

Reply:
I see the relevant code for ts client but not for vuex.

Should I apply theg pattern in ts-client to all?

Reply:
I would vote to do the same in all cases for two reasons, one is to keep consistency with the behaviour between all of them (if possible) and the other because some of those cases print the output path after the generation, it would look odd without the consistency in the paths. It is possible to receive relative and absolute paths so I guess unifying the format is the way to go :)